src/main/java/io/jenkins/plugins/sprp/YAML_FlowDefinition.java

```diff
@@ -125,6 +131,48 @@ public FlowExecution create(FlowExecutionOwner owner, TaskListener listener,
         return new CpsFlowExecution(script, false, owner);
     }
 
+    // In refpecs there will be '+' symbol and some trailing white spaces, we need to remove them.
```

Comment:
the comment is a good improvement, but it doesn't follow the javadoc style that would be preferred in this case